### ISSUE TITLE
Should it also watch the `config/environment.js`?

### ISSUE THREAD

**Issue by ivanvolti** [ebryn/ember-autoserve#1]
Should the addon also restart the server whenever `config/environment.js` is changed?

**Comment by rtablada** [comment]
@ivanvotti The `config/environment.js` file is already watched for you (with the regular `ember serve` command)

**Comment by ivanvolti** [comment]
@rtablada Yep, it's watched, but it doesn't reload all your config this way.

Let's say in your `config/environment.js` you have the content below:

``` js
  module.exports = function(environment) {
    var ENV = {
      modulePrefix: 'my-app',
      environment: environment
    };

    if (environment === 'development') {
      ENV.APP.LOG_RESOLVER = true;
    }

    return ENV;
  };
```

If you try to turn off the `LOG_RESOLVER` by setting it to `false` -- `ENV.APP.LOG_RESOLVER = false;`, ember-cli will reload the config somehow, but nothing will change for you in the browser. You will still have the `LOG_RESOLVER` working. To apply that config change you will have to stop the `ember serve` and start it again.

BTW, some other config changes will be applied correctly, e.g. if you change `ENV.exportApplicationGlobal = false;  // or true` it will work without restarting the `ember serve`
